multimap/src/test/java/org/infinispan/multimap/impl/DistributedMultimapCacheTest.java

```diff
@@ -288,6 +409,58 @@ public void testGetEntry() {
                         }
                   )
       );
+      
+      await(multimapCache.put(EMPTY_KEY, RAMON).thenCompose(r3 -> multimapCache.getEntry(EMPTY_KEY)).thenAccept(v -> {
+         assertTrue(v.isPresent() && v.get().getKey().equals(EMPTY_KEY) && v.get().getValue().contains(RAMON));
+      }));
+   }
+   
+   public void testPutWithNull() {
+      MultimapCache<String, User> multimapCache = getMultimapCacheMember(NAMES_KEY);
```

Comment:
aren't these null tests tests already covered ?

Reply:
They were for Embedded, not for Distributed (which is sad as I wrote these before I moved on to Embedded, then found them there, so I removed mine and just copied these)